solutions/cses-1634.mdx

```diff
@@ -153,4 +153,26 @@ public class cses1634 {
 ```
 </JavaSection>
 
+<PySection>
+```python
+INF = 1000000000
```

Comment:
or just `float('inf')`

Reply:
putting that makes the code tle for some reason 💀

Reply:
add a comment saying that then

Reply:
ok just added